openmetadata-ui/src/main/resources/ui/src/components/TabsLabel/TabsLabel.test.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import TabsLabel from './TabsLabel.component';
+import { TabsLabelProps } from './TabsLabel.interface';
+
+const mockProps: TabsLabelProps = {
+  name: 'Test tab',
+};
+
+describe('TabsLabel component', () => {
+  it('Component should render', async () => {
+    render(<TabsLabel {...mockProps} />);
+
+    expect(await screen.findByText(mockProps.name)).toBeInTheDocument();
+    expect(screen.queryByTestId('count')).not.toBeInTheDocument();
+  });
+
+  it('Count should be visible if provided', async () => {
```

Comment:
Can we cover the test for `isActive` prop as well?

Reply:
We are just passing isActive to the function, so it doesn't have a direct impact on this component

Reply:
Based on that we will have the active badge class right, so can we check for that?

Reply:
so it will be a test of that function, right?

Reply:
Yes, got it, it's a separate function.